homeassistant/components/axis/.translations/en.json

```diff
@@ -0,0 +1,48 @@
+{
+    "config": {
+        "title": "Axis device",
+        "step": {
+            "user": {
+                "title": "Set up Axis device",
+                "data": {
+                    "host": "Host",
+                    "username": "Username",
+                    "password": "Password",
+                    "port": "Port"
+                }
+            },
+            "features": {
+                "title": "Select features from Axis device",
+                "data": {
```

Comment:
Config flows should currently focus on authentication only. We are going to add an options screen in the future. Please remove these options and just include them all.